lib/matplotlib/_types.py

```diff
@@ -0,0 +1,179 @@
+"""
+Style description information that is shared across unrelated classses.
```

Comment:
Since the module is `_types`. Shouldn't we somehow use "types" instead of "style description". "style" seems a bit too specific.

Reply:
New docstring states the real goal of the class better, I hope.